db/seeds.d/11-smart_proxy_features.rb

```diff
@@ -1,5 +1,5 @@
 # Proxy features
-[ "TFTP", "DNS", "DHCP", "Puppet", "Puppet CA", "BMC", "Realm", "Facts" ].each do |input|
+[ "TFTP", "DNS", "DHCP", "Puppet", "Puppet CA", "BMC", "Realm", "Facts", "Logs" ].each do |input|
```

Comment:
Nope, we usually add just seed, as in for example a2523ccf030169eaf170c329e2f68193777ab2cb